vue-app/src/views/RecipientRegistry.vue

```diff
@@ -1,139 +1,134 @@
+                    :src="request.metadata.thumbnailImageUrl"
+                  />
+                </links>
+                {{ request.metadata.name }}
+                <links
+                  v-if="hasProjectLink(request)"
+                  :to="{
+                    name: 'project',
+                    params: { id: request.recipientId },
+                  }"
+                  >-></links
                 >
-                  Remove
+              </div>
+              <details class="project-details">
+                <summary>More</summary>
+
+                <div>
+                  <span
+                    >Transaction hash
+                    <button
+                      class="button-copy"
+                      @click="copyAddress(request.transactionHash)"
+                    >
+                      <img src="@/assets/copy.svg" />
+                    </button>
+                  </span>
+                  <code>{{ request.transactionHash }}</code>
+                </div>
+                <div>
+                  <span
+                    >Project ID
+                    <button
+                      class="button-copy"
+                      @click="copyAddress(request.recipientId)"
+                    >
+                      <img src="@/assets/copy.svg" />
+                    </button>
+                  </span>
+                  <code>{{ request.recipientId }}</code>
                 </div>
+                <div>
+                  <span
+                    >Recipient address
+                    <button
+                      class="button-copy"
+                      @click="copyAddress(request.recipient)"
+                    >
+                      <img src="@/assets/copy.svg" />
+                    </button>
+                  </span>
+                  <code>{{ request.recipient }}</code>
+                </div>
+              </details>
+            </td>
+            <td>{{ request.type }}</td>
+            <td>
+              <template v-if="hasProjectLink(request)">
+                <links
+                  :to="{
+                    name: 'project',
+                    params: { id: request.recipientId },
+                  }"
+                >
+                  {{ request.status }}
+                </links>
+              </template>
+              <template v-else>
+                {{ request.status }}
+              </template>
+            </td>
+            <td>
+              <div class="actions" v-if="isUserConnected">
+                <!-- TODO: to implement this feature, it requires to send a baseDeposit (see contract)
+              <div
+                class="btn-warning"
+                @click="remove(request)"
+                v-if="isExecuted(request)"
+              >
+                Remove
+              </div> -->
                 <div
                   class="icon-btn-approve"
+                  v-if="
+                    (isOwner || (!isOwner && !isChallengePeriod)) &&
```

Comment:
Again I believe `isChallengePeriod` will vary be recipient depending on when they applied.